src/adapters/lokijs/index.js

```diff
@@ -109,6 +109,9 @@ export default class LokiJSAdapter implements DatabaseAdapter {
     this._dbName = dbName
 
     if (process.env.NODE_ENV !== 'production') {
+      invariant('password' in options,
```

Comment:
It would be bad if someone didn't realize that encryption wasn't supported, so best to safeguard.